packages/solutions/module-tools/src/builder/feature/swc.ts

```diff
@@ -40,7 +40,7 @@ const getSwcTarget = (target: string): JscTarget => {
   return 'es2022';
 };
 
-export const swcTransform = (userTsconfig: ITsconfig) => ({
+export const swcTransform = (userTsconfig: ITsconfig, tsVersion: number) => ({
```

Comment:
Is it OK to add a parameter like that?

Reply:
This function is an internal, seems acceptable.